infrastructure/stacks/development-pipeline/security_groups.tf

```diff
@@ -7,11 +7,13 @@ resource "aws_security_group" "uec_dos_int_int_test_sg" {
   }
 }
 
-resource "aws_security_group_rule" "allow_all_out" {
+#tfsec:ignore:aws-vpc-no-public-egress-sgr
+resource "aws_security_group_rule" "allow_https_out" {
   type              = "egress"
-  from_port         = 0
-  to_port           = 0
-  protocol          = "-1"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
   cidr_blocks       = ["0.0.0.0/0"]
```

Comment:
Does this CIDR need to be defined ?

Reply:
You have to set either the CIDR, or another variable which I could get any others to work

Reply:
`Error: One of ['cidr_blocks', 'ipv6_cidr_blocks', 'self', 'source_security_group_id', 'prefix_list_ids'] must be set to create an AWS Security Group Rule`
I tried self which didn't seem to work